src/main/groovy/com/ofg/twitter/place/extractor/PlacesExtractor.groovy

```diff
@@ -47,6 +54,22 @@ class PlacesExtractor {
         }
         return Optional.empty()
     }
+}
 
-
+//TODO: Remove when integrated with micro-infra-spring (and released)
+@CompileStatic
+@PackageScope
+class CorrelationIdUpdater2PoC {
+    static <T> Closure<T> closureWithId(Closure<T> closure) {
```

Comment:
Can't we just wait for micro-infra-spring release and avoid this class altogether? I guess it's a copy from micro-infra?